cookbooks/bcpc/recipes/mysql.rb

```diff
@@ -56,7 +56,7 @@
   execute "percona-preseed-#{preseed_item}" do
     command 'echo "percona-xtradb-cluster-server-5.6 ' \
       "percona-xtradb-cluster-server/#{preseed_item} " \
-      "password #{root_password}\" | debconf-set-selections"
+      "password \"#{root_password}\" | debconf-set-selections"
```

Comment:
Yep.  I think it may have been silently failing in the past, not sure how it ever worked with the missing quote mark.